src/pages/detail/tabs/notices/GroupNoticesTab.tsx

```
@@ -2,23 +2,20 @@ import { Suspense } from "react";
 
 import CategorizedNotices from "@/components/categorizedNotices/CategorizedNotices";
 import Loading from "@/components/loading/Loading";
-import { useParams } from "react-router-dom";
-import axios from "axios";
-import useSWR from "swr";
+
 interface GroupNoticesTabProps {
   searchParams?: { page: string };
 }
 
 const GroupNoticesTab = ({}: GroupNoticesTabProps) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)